forJIS7/src/main/java/JavaHomework/Lectures_17.Stream/secondPart/Service/UserService.java

```diff
@@ -0,0 +1,55 @@
+package JavaHomework.Lectures_17.Stream.secondPart.Service;
+
+import JavaHomework.Lectures_17.Stream.secondPart.Model.User;
+import JavaHomework.Lectures_17.Stream.secondPart.Model.UserStatus;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static JavaHomework.Lectures_17.Stream.secondPart.Repository.UserRepository.*;
+
+public class UserService implements Service {
+
+    @Override
+    public void showUserList() {
+        userList
+                .forEach(System.out::println);
+    }
+
+    @Override
+    public List<User> changeIsActive() {
+        return userList.stream()
+                .peek(it->it.setIsActive(true))
+                .collect(Collectors.toList());
+    }
+
+    @Override
+    public List<User> changeUserStatusGuestToUser() {
+         return userList.stream()
+                .peek(it->it.setUserStatus(UserStatus.USER))
+                 .collect(Collectors.toList());
+    }
+
+    @Override
+    public List<User> checkUserStatus() {
+        return userList.stream()
+                .filter(it -> it.getUserStatus().equals(UserStatus.GUEST))
+                .collect(Collectors.toList());
```

Comment:
As I see, you took Users, found who has status GUEST but didn't take the filtered list. If you have the filtered list with status GUEST, why you didn't use it in the method changeUserStatusGuestToUser?

If you don't take the filtred list, for what you filtred them?